backend/guides/code.md

```diff
@@ -0,0 +1,159 @@
+# Coding
+
+## Source code
+
+### Naming above all
+
+The source code you write is intended for only two audiences
+
+- other people,
+- compiler/interpreter.
+
+The compiler doesn't care about the naming, doesn't have to understand the semantics, therefore the readability and understandability are the most important aspects above all.
+
+It's exactly like when two people from different countries trying to express themselves with language the other doesn't understand. Or even worse, they speak the same language and each of them are using different naming for the same things.
+
+When you call something 'banana', it should be banana and not an orange.
+
+### Learn to love old code
+
+If it works, there is no need to refactor it. Refactoring a code that is old but checks out can be done over and over and forever.
+
+Do you need to add functionality? Compose the code.
+
+### Minimal, conscious, self-documenting
+
+1. Less code is oftentimes better, than more code, if it does not sacrifice readability or comprehension. Smaller codebase is more maintainable and shorter snippets are faster to review and understand.
+2. Avoid using one-liners if it hurts the readability
+
+```js
+// BAD
+const articles = getArticles()
+const filteredArticles = []
+for (const article of articles) {
+    if (isValid(article)) {
+        filteredArticles.push(article)
+    }
+}
+// GOOD
+const filteredArticles = getArticles().filter(isValid)
+```
+
+As seen in the example, limiting local variables and imperative constructs usually leads to more readable code, that does not need comments and is easier to follow.
+
+This practice is sometimes referred to as [💋 KISS](https://en.wikipedia.org/wiki/KISS_principle)
+
+## Prefer libraries
+
+Always prefer libraries above your code - you transfer the responsibility to the libraries, that usually are well tested
+
+Get to know our libs!
```

Comment:
:question: what about `enmail`, `crudella` ?

Reply:
`enmail` is there, L54. Should we add crudella?

Reply:
sorry, i didnt see enmail :( 

crudella .. i dont know, authist is there but its under development yet